cli/constants.py

```diff
@@ -5,6 +5,7 @@
 BATCH_ID_REGEXP = r'^[0-9a-f]{128}$'
 PUBLIC_KEY_REGEXP = r'^[0-9a-f]{66}$'
 PRIVATE_KEY_REGEXP = r'^[a-f0-9]{64}$'
+PUBLIC_KEY_REGEXP = r'^[0-9a-f]{66}$'
```

Comment:
Delete the superfluous same `PUBLIC_KEY_REGEXP = r'^[0-9a-f]{66}$'`.

Reply:
@Alladin9393, done.